appveyor.yml

```diff
@@ -6,7 +6,7 @@ build:
 
 build_script:
  - cmake -P ci-build.cmake
- - cmake --build .
+ - cmake --build . --config Debug --target INSTALL
```

Comment:
please drop the `--target INSTALL` part here as it spams the build logs. otherwise this looks good to me.